### PR TITLE
Test SNDev branch

### PR DESCRIPTION
**Pull request by ssa0551**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,6 @@
 Study Ninja App Repository
 ## Initiative1 
 
-### User Stories
\ No newline at end of file
+### User Story1
```
Comment (Gurpreet2804) [inline README.md:5]: update this

**Review by Gurpreet2804 — COMMENTED** [review]
please fix the comments

File: README.md (hunk 1 above)
Comment (ssa0551) [reply README.md:5]: Comments updated, please review

**Review by Gurpreet2804 — APPROVED** [review]
looks good

**Pull request merged**